tests/migrations/test_autodetector.py

```diff
@@ -47,6 +47,22 @@ class AutodetectorTests(TestCase):
         ("id", models.AutoField(primary_key=True)),
         ("name", models.CharField(max_length=200, default='Ada Lovelace')),
     ])
+    author_date_of_birth_auto_now = ModelState("testapp", "Author", [
+        ("id", models.AutoField(primary_key=True)),
+        ("date_of_birth", models.DateField(auto_now=True)),
```

Comment:
Can you combine the model states to have 3 fields: `DateField`, `DateTimeField`, and `TimeField`.